subsys/bluetooth/controller/Kconfig

```diff
@@ -882,6 +882,15 @@ config BT_CTLR_USER_EVT_RANGE
 	  Number of event types reserved for proprietary use. The range
 	  is typically used when BT_CTLR_USER_EXT is in use.
 
+config BT_CTLR_USER_TICKER_ID_RANGE
+	int "Range of ticker id constants reserved for proprietary ticker nodes"
+	depends on BT_CTLR_USER_EXT
+	default 0
+	range 0 10
```

Comment:
Do we need to add a range limitation on this?

Reply:
> Do we need to add a range limitation on this?

No we don't need the range.